src/components/application_manager/src/smart_object_keys.cc

```diff
@@ -561,6 +561,7 @@ const char* capabilities = "capabilities";
 const char* speech_capabilities = "speechCapabilities";
 const char* prerecorded_speech_capabilities = "prerecordedSpeechCapabilities";
 const char* preset_bank_capabilities = "presetBankCapabilities";
+const char* on_screen_present_available = "onScreenPresetsAvailable";
```

Comment:
Typo `present`

Reply:
@ydementieiev Done in 8d263aa14997dd8cb47c098402954deb6ae0f106